engine/polyglot-api/src/main/scala/org/enso/polyglot/Suggestion.scala

```diff
@@ -42,6 +42,14 @@ object Suggestion {
   sealed trait Kind
   object Kind {
 
+    def apply(suggestion: Suggestion): Kind =
+      suggestion match {
+        case _: Atom     => Atom
+        case _: Method   => Method
+        case _: Function => Function
+        case _: Local    => Local
```

Comment:
Basically, a `Local` value is a `Function` without arguments. When designing suggestions I thought we can distinguish between them because `Local` is cached by the interpreter and behaves like a value.
Removing `Local` would be a substantial change in API. I want to create a separate issue if we decide to change that. This way it would be easier for the IDE team to track the changes.

Reply:
You misunderstand me. With locals, we can have a few cases:
- A sugared function: `f a b = ...`
- A lambda: `f = a -> b -> ...`
- A value: `f = a`

Do suggestions for locals distinguish between the latter two cases?

Reply:
No,
- a sugared function `f a b = ...` is a `Function`
- a lambda `f = a -> ...` is also a `Function`
- a value `f = a` is a `Local`

Reply:
Okay, brilliant. That's what I was hoping for.

Reply:
Out of curiosity: what happens with `f = g` when `g` is a `Function?